src/liblelantus/sigmaextended_verifier.h

```diff
@@ -43,11 +43,18 @@ class SigmaExtendedVerifier{
             const SigmaExtendedProof& proof,
             const Scalar& x,
             const std::vector<Scalar>& f_) const;
+
+    void compute_fis(const Scalar& f_i, int j, const std::vector<Scalar>& f) const;
+    void compute_batch_fis(const Scalar& f_i, int j, const std::vector<Scalar>& f, const Scalar& y, Scalar& e) const;
+
 private:
     GroupElement g_;
     std::vector<GroupElement> h_;
     uint64_t n;
     uint64_t m;
+    mutable Scalar* ptr;
+    mutable Scalar* start_ptr;
+    mutable Scalar* end_ptr;
 };
```

Comment:
not a nice way to do it. Please consider moving these to `compute_fis`/`compute_batch_fis` arguments